requirements.txt

```diff
@@ -0,0 +1,51 @@
+black @ file:///croot/black_1712698501521/work
+Bottleneck @ file:///croot/bottleneck_1707864210935/work
+Brotli @ file:///croot/brotli-split_1714483155106/work
+certifi @ file:///croot/certifi_1707229174982/work/certifi
+charset-normalizer @ file:///tmp/build/80754af9/charset-normalizer_1630003229654/work
```

Comment:
Why do you need a `requirements.txt`.? We will just use conda `env.yml` to store all the _dependencies_.